### PR TITLE
Drop unused install keyword and load TS workflows with utf-8 encoding

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

**Comment by mvdbeek** [comment]
Test is failing because we need to merge forward #7418

File: lib/galaxy/webapps/galaxy/api/workflows.py [hunk 1]
```diff
@@ -302,8 +303,7 @@ def create(self, trans, payload, **kwd):
             if not os.path.exists(installed_repository_file):
                 raise exceptions.MessageException("Repository file '%s' not found.")
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/workflows.py]:
```suggestion
                raise exceptions.RequestParameterInvalidException("Workflow file '%s' not found" % installed_repository_file)
```

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
@mvdbeek I've merged #7418 forward to dev if you want to rebase this PR.

File: lib/galaxy/webapps/galaxy/api/workflows.py (hunk 1 above)
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/workflows.py]: This got lost in the rebase:
```suggestion
                raise exceptions.RequestParameterInvalidException("Workflow file '%s' not found" % installed_repository_file)
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**